pages/contact.vue

```diff
@@ -1,15 +1,109 @@
 <template>
-  <div class="white-wrapper">
-    Contact Page
+  <div class="background-section">
+    <div class="title">
+      <h1 class="cursive">Contact & Directions</h1>
+      <h4><font-awesome-icon icon="phone"/> (617) 924-9595</h4>
+    </div>
+    <div class="contact-section">
+        <div class="outside-image">
```

Comment:
Looks like there's extra spacing here. Let's move the next few lines back.

Reply:
Removed the spacing